composeApp/src/commonMain/kotlin/design-system/PrimaryMainButton.kt

```diff
@@ -0,0 +1,78 @@
+package design
```

Comment:
Wrong package. AS can help you in this fix.

Reply:
Done